### ISSUE TITLE
#![deny(intra_doc_link_resolution_failure)]

### ISSUE THREAD

**Issue by jamwaffles** [jamwaffles/embedded-graphics#186]
From [this tweet](https://twitter.com/tomaka17/status/1176017851410526208):

> Some #rustlang thing I've recently embraced:
> 
> - Add `#![deny(intra_doc_link_resolution_failure)]` at the crate root.
> - Add `cargo doc --document-private-items` to your CI.
> - Put links absolutely everywhere in your docs.
> 
> This way, CI detects if a code change breaks some docs.

Currently, this project uses [linkchecker](https://wummel.github.io/linkchecker/). Is there a place for both/just Cargo's checking to be included?

**Comment by jamwaffles** [comment]
`intra_doc_link_resolution_failure` is nightly-only and only works with intra-doc links, not standard Markdown links, so is also blocked by rust-lang/rust#43466 as is #100.

**Comment by jamwaffles** [comment]
Closing in favour of #100
